staging/src/k8s.io/kubectl/pkg/cmd/scale/scale.go

```diff
@@ -211,6 +211,10 @@ func (o *ScaleOptions) RunScale() error {
 	// to show invalid resources to the user.
 	infos, infoErr := r.Infos()
 
+	if infoErr != nil {
```

Comment:
`infoErr` should not be returned as shortcut because it will iterate over the other resources. I recommended something different in https://github.com/kubernetes/kubectl/issues/1666#issuecomment-2410544964

/hold

Reply:
got this.